source/js/mozfest-event-handlers.js

```diff
@@ -0,0 +1,59 @@
+const bindHomeBannerHandlers = () => {
+  let homepageBanner = document.querySelector(
+    "#view-mozfest-home #hero .banner"
+  );
+
+  if (!homepageBanner) {
+    return;
+  }
+
+  let video = homepageBanner.querySelector("video.banner-video");
+  let pauseButton = homepageBanner.querySelector(
+    ".btn-video-control.btn-pause"
+  );
+  let playButton = homepageBanner.querySelector(".btn-video-control.btn-play");
+
+  if (video && pauseButton && playButton) {
+    let showVideoControls = () => {
+      const classToToggle = `d-none`;
+      let autoplayStarted =
+        video.played.length == 0 && video.readyState >= video.HAVE_FUTURE_DATA;
+
+      if (video.readyState < video.HAVE_FUTURE_DATA) {
+        // don't show any video controls as the video isn't ready to play
+        playButton.classList.add(classToToggle);
+        pauseButton.classList.add(classToToggle);
+      } else if (!video.paused || autoplayStarted) {
+        playButton.classList.add(classToToggle);
+        pauseButton.classList.remove(classToToggle);
+      } else {
+        pauseButton.classList.add(classToToggle);
+        playButton.classList.remove(classToToggle);
+      }
+    };
+
+    pauseButton.addEventListener(`click`, () => {
+      video.pause();
+      showVideoControls();
+    });
+
+    playButton.addEventListener(`click`, () => {
+      video.play();
+      showVideoControls();
+    });
+
+    showVideoControls();
+
+    video.addEventListener(
+      `canplay`,
+      () => {
+        showVideoControls();
+      },
+      false
```

Comment:
`useCapture` is basically an irrelevant option in modern browsing, and defaults to `false` anyway, so there shouldn't be a reason to specify it.